notebook/static/notebook/js/actions.js

```diff
@@ -545,13 +552,13 @@ define(function(require){
         },
     };
 
-    // private stuff that prepend `.ipython` to actions names
+    // private stuff that prepend `jupyter-notebook:` to actions names
     // and uniformize/fill in missing pieces in of an action.
     var _prepare_handler = function(registry, subkey, source){
-        registry['ipython.'+subkey] = {};
-        registry['ipython.'+subkey].help = source[subkey].help||subkey.replace(/-/g,' ');
-        registry['ipython.'+subkey].help_index = source[subkey].help_index;
-        registry['ipython.'+subkey].icon = source[subkey].icon;
+        registry['jupyter-notebook:'+subkey] = {};
+        registry['jupyter-notebook:'+subkey].help = source[subkey].help||subkey.replace(/-/g,' ');
+        registry['jupyter-notebook:'+subkey].help_index = source[subkey].help_index;
+        registry['jupyter-notebook:'+subkey].icon = source[subkey].icon;
```

Comment:
We are looking ahead to the many separate plugins/components situation where there will be lots of jupyter-components (jupyter-terminal, jupyter-filebrowser) in addition to third party ones (bloomberg-this, continuum-that, etc.). We will need this level of namespacing in both KB shortcuts, CSS, etc.

Also, in a single page app, it is not clear that the "notebook" component is the "builtin" one - someone may use the Jupyter web app and never run a notebook.